grafana/resource_annotation_test.go

```diff
@@ -164,31 +164,24 @@ resource "grafana_annotation" "test_with_dashboard_id" {
 }
 
 func testAnnotationConfigWithPanelID(text string) string {
+	panelID := 123
+
 	return fmt.Sprintf(`
 resource "grafana_dashboard" "test_with_panel_id" {
   config_json = <<EOD
 {
   "title": "%s",
 	"panels": [{
 		"name": "%s",
-		"id": 123
+		"id": %d
 	}]
 }
 EOD
 }
 
-data "grafana_dashboard" "test_with_panel_id" {
-	dashboard_id = grafana_dashboard.test_with_panel_id.dashboard_id
-}
-
-locals {
-  dashboard_json = jsondecode(data.grafana_dashboard.test_with_panel_id.config_json)
-  panel_id       = local.dashboard_json.panels[0].id
-}
-
 resource "grafana_annotation" "test_with_panel_id" {
     text     = "%s"
-		panel_id = local.panel_id
+		panel_id = %d
 }
-`, text, text, text)
+`, text, text, panelID, text, panelID)
```

Comment:
Note: this change was added to make annotation resource tests pass. They were failing with the following error:
```
=== RUN   TestAccAnnotation_basic
    resource_annotation_test.go:24: Step 9/9 error running import: exit status 1
        Error writing state file: Failed to serialize resource instance in state: Instance data.grafana_dashboard.test_with_panel_id has status ObjectPlanned, which cannot be saved in state.
```